okapi-common/src/test/java/org/folio/okapi/common/PauseResumeTest.java

```diff
@@ -37,7 +37,7 @@ private void myStreamHandle2(RoutingContext ctx) {
         return;
       }
       req.result().end();
-      req.result().onComplete(res -> {
+      req.result().response(res -> {
```

Comment:
This is ok.
Replacing HttpClient by WebClient allows for more concise code and would have avoided this code change.
You don't need to replace HttpClient by WebClient before merging this pull request.
```
    WebClient
        .create(vertx)
        .post(PORT, "localhost", "/test1")
        .send()
        .onSuccess(res -> {
          if (ctx.request().isEnded()) {
            ctx.response().end("OK2"); // Vert.x 3.6 series
          } else {
            ctx.request().endHandler(y -> {
              ctx.response().end("OK2");
            });
            ctx.request().resume();
          }
        }).onFailure(cause -> {
          ctx.response().setStatusCode(500);
          ctx.response().end(cause.getMessage());
        });
```